client/src/main/java/io/pravega/client/netty/impl/ConnectionFactoryImpl.java

```diff
@@ -124,6 +124,7 @@ private int getNumThreads(Integer numThreadsInPool) {
                 if (Strings.isNullOrEmpty(clientConfig.getTrustStore())) {
                     sslCtxFactory = sslCtxFactory.trustManager(FingerprintTrustManagerFactory
                                                       .getInstance(FingerprintTrustManagerFactory.getDefaultAlgorithm()));
+                    log.warn("Done setting the sslCtxFactory to FingerprintTrustManagerFactory");
```

Comment:
This `debug` level, but I think we can remove this altogether, unless you think such a message can help in the future with debugging. If so, let's put down a message that is less cryptic.

Reply:
Fixed. I want to keep it, as it can help debug SSL issues.